packages/plugin-cloudflare/scripts/postinstall.js

```diff
@@ -90,6 +91,20 @@ export default async function install() {
   }
 }
 
+function versionIsGreaterThan(versionA, versionB) {
```

Comment:
Yeah, that's why I wanted to remove `semver`, we know what we are comparing so it should be safe

Reply:
it's not trivial to add tests to a post-install script, because I can't add tests inside the `script` folder and I can't test from the `src` folder because I "can't" import any file from `script` there 🤔